syntax = "proto3";

package yandex.mdb.cms.v1;
option go_package = "a.yandex-team.ru/cloud/mdb/cms/api/grpcapi/v1;cmsv1";

import "cloud/mdb/cms/api/grpcapi/v1/instance_operation.proto";

service InstanceOperationService {
    rpc Get(GetInstanceOperationRequest) returns (InstanceOperation) {}
    rpc List(ListInstanceOperationsRequest) returns (ListInstanceOperationsResponse) {}
}

message GetInstanceOperationRequest {
    string id = 1; // must be valid UUIDv4
}

message ListInstanceOperationsRequest {

}

message ListInstanceOperationsResponse {
    repeated InstanceOperation operations = 1;
}
